### PR TITLE
Adds cime testmods and introduces new directory structure

### PR DESCRIPTION
**Pull request by singhbalwinder**

### CODE REVIEW

File: components/eamxx/cime_config/namelist_defaults_scream.xml [hunk 1]
```diff
@@ -252,14 +252,14 @@ be lost if SCREAM_HACK_XML is not enabled.
...
+    <mam4_oxid_file_name  hgrid="ne4np4" type="file" doc="">${DIN_LOC_ROOT}/atm/scream/mam4xx/invariants/ne4pg2/oxid_ne4pg2_L26_1850-2015_c20240821.nc</mam4_oxid_file_name>
+    <mam4_linoz_chlorine_file type="file" doc=""> ${DIN_LOC_ROOT}/atm/scream/mam4xx/linoz/Linoz_Chlorine_Loading_CMIP6_0003-2017_c20171114.nc</mam4_linoz_chlorine_file>
+    <mam4_chlorine_loading_ymd type="integer" doc=""> 20100101 </mam4_chlorine_loading_ymd>
+    <mam4_rsf_file type="file" doc=""> ${DIN_LOC_ROOT}/atm/scream/mam4xx/photolysis/RSF_GT200nm_v3.0_c080811.nc</mam4_rsf_file>
+    <mam4_xs_long_file type="file" doc=""> ${DIN_LOC_ROOT}/atm/scream/mam4xx/photolysis/temp_prs_GT200nm_JPL10_c130206.nc</mam4_xs_long_file>
+    <mam4_bc_a4_verti_emiss_file_name hgrid="ne4np4" type="file" doc="">${DIN_LOC_ROOT}/atm/scream/mam4xx/emissions/ne4pg2/elevated/cmip6_mam4_bc_a4_elev_ne4pg2_2010_clim_c20240821.nc </mam4_bc_a4_verti_emiss_file_name>
+    <mam4_so2_verti_emiss_file_name hgrid="ne4np4" type="file" doc="">${DIN_LOC_ROOT}/atm/scream/mam4xx/emissions/ne4pg2/elevated/cmip6_mam4_so2_elev_ne4pg2_2010_clim_c20240821.nc</mam4_so2_verti_emiss_file_name>
```
Comment (odiazib) [inline components/eamxx/cime_config/namelist_defaults_scream.xml:262]: You also need to add the vertical emissions for the other seven species.

File: components/eamxx/tests/single-process/mam/aero_microphys/input.yaml [hunk 2]
```diff
@@ -11,21 +11,21 @@ time_stepping:
...
-   mam4_num_a2_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_num_a2_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_num_a4_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_num_a4_elev_ne2np4_2010_clim_c20190821_OD.nc
-   mam4_soag_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/physprops/cmip6_mam4_soag_elev_ne2np4_2010_clim_c20190821_OD.nc
+   mam4_linoz_file_name : ${SCREAM_DATA_DIR}/mam4xx/linoz/ne2np4/linoz1850-2015_2010JPL_CMIP6_10deg_58km_ne2np4_c20240724.nc
+   mam4_oxid_file_name : ${SCREAM_DATA_DIR}/mam4xx/invariants/ne2np4/oxid_1.9x2.5_L26_1850-2015_ne2np4L72_c20240722_OD.nc
+   mam4_linoz_chlorine_file : ${SCREAM_DATA_DIR}/mam4xx/linoz/Linoz_Chlorine_Loading_CMIP6_0003-2017_c20171114.nc
+   mam4_chlorine_loading_ymd : 20100101   
+   mam4_rsf_file : ${SCREAM_DATA_DIR}/mam4xx/photolysis/RSF_GT200nm_v3.0_c080811.nc
+   mam4_xs_long_file : ${SCREAM_DATA_DIR}/mam4xx/photolysis/temp_prs_GT200nm_JPL10_c130206.nc
+   mam4_so2_verti_emiss_file_name : ${SCREAM_DATA_DIR}/mam4xx/emissions/ne2np4/elevated/cmip6_mam4_so2_elev_ne2np4_2010_clim_c20240726.nc
```
Comment (odiazib) [inline components/eamxx/tests/single-process/mam/aero_microphys/input.yaml:20]: These files also need to be added to the namelist file.

**Review by odiazib — APPROVED** [review]

File: components/eamxx/cime_config/namelist_defaults_scream.xml (hunk 1 above)
Comment (singhbalwinder) [reply components/eamxx/cime_config/namelist_defaults_scream.xml:262]: Yes, I will do that in the next iteration as they are only needed for the CIME test.

File: components/eamxx/tests/single-process/mam/aero_microphys/input.yaml (hunk 2 above)
Comment (singhbalwinder) [reply components/eamxx/tests/single-process/mam/aero_microphys/input.yaml:20]: Yes, I will do that in the next iteration.

**Pull request merged**